examples/job/create_and_submit_job.py

```diff
@@ -11,25 +11,28 @@
 # 
 # ## Import packages
 
-# In[2]:
-
+# In[1]:
 
-import json
 
-from exabyte_api_client.endpoints.jobs import JobEndpoints
-from exabyte_api_client.endpoints.materials import MaterialEndpoints
-from exabyte_api_client.endpoints.workflows import WorkflowEndpoints
+from IPython.display import JSON
+import os
+import sys
 
-# Import settings file
-import os,sys
+# Import settings and utils file
 module_path = os.path.abspath(os.path.join('..'))
 if module_path not in sys.path: sys.path.append(module_path)
 from settings import ENDPOINT_ARGS, ACCOUNT_ID
+from utils import ensure_packages_are_installed
+ensure_packages_are_installed()
+
+from exabyte_api_client.endpoints.jobs import JobEndpoints
+from exabyte_api_client.endpoints.materials import MaterialEndpoints
+from exabyte_api_client.endpoints.workflows import WorkflowEndpoints
 
 
 # ## Initialize the endpoints
 
-# In[15]:
+# In[2]:
```

Comment:
We should do some postprocessing to avoid dealing with changing numbers in the execution order.